tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -112,6 +112,10 @@ def batch_norm(inputs,
                scale=False,
                epsilon=0.001,
                activation_fn=None,
+               beta_initializer=init_ops.zeros_initializer,
```

Comment:
can you replace this with a dict of initializers which would default to their corresponding value if they key is not present, similar to variables_collections.